### DISCUSSION TITLE
create a form page to request manual award

### DISCUSSION THREAD

**Discussion by meleu** [RetroAchievements/RAWeb#711] [Ideas]
**Note:** this is not a feature to be implemented soon. Maybe we need the notifications system to tag admins before implementing this.

Our #help-me channel at discord is turning into a #gimme-the-cheevo channel, and we constantly have to explain that users must provide more info.

Then I thought in a form with the following fields:

- the ticket for the buggy cheevo (not necessary if it was a network problem)
- the achievement ID
- a proof (could be a link to an image/video)
- comments

**Comment by kdecks** [comment]
Great idea

**Comment by Jamiras** [comment]
We have instructions for this in the documentation: https://docs.retroachievements.org/FAQ/#how-can-i-get-credit-for-an-achievement-i-earned-but-wasnt-awarded

The instructions tell the user to create a ticket (unless it's network related), then post in discord or send a message to RAdmin.

**Reply by meleu** [reply]
yeah, but the idea of having a form would be useful to request all mandatory information with no need to point users to read documentation.

**Comment by LiquifiedSnow** [comment]
New idea for manual unlock process - Page on the website similar to the ticket system or a branch of the tickets where you could put in a manual unlock request.

When creating a ticket, you'd be prompted to select yes or no to requiring a manual unlock request. If you choose yes it would ask you for a link to the photo/video proof and it would then generate a ticket and a manual unlock request.

Manual unlock page would look similar to tickets with an option for the developer to verify if they want and a button for admin's to approve and automatically award. 

Requests would be sorted like ticket page where you could view open, denied, approved and verified requests.

- Open - Show current open manual unlock requests just like open tickets
- Denied - Show all manual unlock's that have been denied for whatever reason
- Closed - Show manual unlocks that were closed by the requester
- Verified - Show manual unlocks that were "Verified" by the developer of the set to let admin's know it's good to unlock and they are aware/have fixed the issue
- Approved - Show  manual unlocks that have been approved and given by an admin

Mock Ups

![Manual Unlocks Manager](https://user-images.githubusercontent.com/6635720/194162709-43d7ad64-e02e-410e-aca8-152ea56e448a.jpg)
![Manual Unlocks Manager2](https://user-images.githubusercontent.com/6635720/194162715-ea371671-b0b1-4625-b65f-35c1b5e92071.jpg)
